### ISSUE TITLE
关于Triple Classification 测试

### ISSUE THREAD

**Issue by penelope24** [thunlp/OpenKE#49]
我阅读了readme里边的介绍，理解了这个测试通过计算某个三元组头尾节点之间的dissimilarity来衡量该三元组是否正确。
我困惑的地方在于这个测试的结果该如何理解，比如说我在自己的数据上跑的结果在执行Triple Classification测试的时候结果特别高（达到了0.99），那么我改如何理解这个测试结果呢？这个测试结果可以告诉我什么呢。
小白望赐教，O(∩_∩)O谢谢！

**Comment by ShulinCao** [comment]
triple classification 可以告诉你，你的模型能不能比较准确地地区分出一个三元组正确还是错误。比如0.99可以大致理解为100个三元组中只有一个被你的模型误判了（错误的被分成正确或正确的被分成错误）

**Comment by penelope24** [comment]
@ShulinCao 非常感谢！triple classification我看代码貌似是对validation set来进行操作的，那是否可以理解为起着验证集的作用呢。

**Comment by ShulinCao** [comment]
不是。triple classification需要找到一个阈值，score小于阈值的三元组判断为正确，反之错误。validation set的作用是找到最佳阈值，即让validation set准确率高的阈值是最佳阈值，这个最佳阈值会被用在测试集。

**Comment by penelope24** [comment]
@ShulinCao 谢谢耐心解答！
